pkgs/development/python-modules/skein/skeinjar.nix

```diff
@@ -1,21 +1,19 @@
-{ callPackage, stdenv, maven, src, version }:
+{ fetchurl, unzip, stdenv, version, jarHash }:
 
-let
-  skeinRepo = callPackage ./skeinrepo.nix { inherit src version; };
-in
 stdenv.mkDerivation rec {
   pname = "skein.jar";
+  inherit version;
 
-  inherit version src;
+  src = fetchurl {
+    url = "https://files.pythonhosted.org/packages/py2.py3/s/skein/skein-${version}-py2.py3-none-any.whl";
+    hash = jarHash;
+  };
 
-  nativeBuildInputs = [ maven ];
-
-  buildPhase = ''
-    mvn --offline -f java/pom.xml package -Dmaven.repo.local="${skeinRepo}" -Dskein.version=${version} -Dversion=${version}
-  '';
+  dontUnpack = true;
+  dontBuild = true;
```

Comment:
Are these causing issues? Otherwise please leave them activated.

Reply:
`dontUnpack` is needed. `fetchURL` will try to extract the whl file and fail. Removed dontBuild.